app/services/case_filter/exemption_filter.rb

```diff
@@ -0,0 +1,90 @@
+module CaseFilter
+  class ExemptionFilter < CaseMultiChoicesFilterBase
+
+    def self.identify
```

Comment:
Here it may be better to use `class << self` and bunch all the class methods together as there are a few of them in this class.